packages/controls/src/jest-setup.ts

```diff
@@ -0,0 +1,3 @@
+jest.mock('uuid', () => ({
```

Comment:
Is there a way this mock can be confined to the setup of the shape tests, rather than a global setup?

Reply:
We have it globally mocked in `runtime`, and IMHO this is a reasonable default. You could probably come up w/ a test use case where you want more flexibility, but it can be overridden locally then.

Reply:
This seems like it could cause issue with tests that expect UUIDs to be different, no? Is the purpose here to make it so snapshots don't fail whenever we're using UUIDs? If so, it would be great if we could address this with some sort of seed value that makes tests deterministic